### PR TITLE
Revert "app status"

### PR DESCRIPTION
**Pull request by calellowitz**
Reverts dimagi/commcare-hq#15514
@benrudolph as per our conversation

### CODE REVIEW

File: corehq/pillows/mappings/user_mapping.py [hunk 1]
```diff
@@ -1,19 +1,28 @@
...
+                  "yyyy-MM-dd'T'HH:mm:ss.SSSZZ",
+                  "yyyy-MM-dd'T'HH:mm:ss",
+                  "yyyy-MM-dd' 'HH:mm:ss",
+                  "yyyy-MM-dd' 'HH:mm:ss.SSSSSS",
+                  "mm/dd/yy' 'HH:mm:ss"],
  'dynamic': False,
  'properties': {'CURRENT_VERSION': {'type': 'string'},
                 'base_doc': {'type': 'string'},
-                'created_on': {'format': DATE_FORMATS_STRING,
+                'created_on': {'format': "yyyy-MM-dd||yyyy-MM-dd'T'HH:mm:ssZZ||yyyy-MM-dd'T'HH:mm:ss.SSSSSS||yyyy-MM-dd'T'HH:mm:ss.SSSSSS'Z'||yyyy-MM-dd'T'HH:mm:ss'Z'||yyyy-MM-dd'T'HH:mm:ssZ||yyyy-MM-dd'T'HH:mm:ssZZ'Z'||yyyy-MM-dd'T'HH:mm:ss.SSSZZ||yyyy-MM-dd'T'HH:mm:ss||yyyy-MM-dd' 'HH:mm:ss||yyyy-MM-dd' 'HH:mm:ss.SSSSSS||mm/dd/yy' 'HH:mm:ss",
```
Comment (dimagimon) [inline corehq/pillows/mappings/user_mapping.py]: E501 line too long (346 > 115 characters)

File: corehq/pillows/mappings/user_mapping.py [hunk 2]
```diff
@@ -1,19 +1,28 @@
...
                                'type': 'date'},
-                'date_joined': {'format': DATE_FORMATS_STRING,
+                'date_joined': {'format': "yyyy-MM-dd||yyyy-MM-dd'T'HH:mm:ssZZ||yyyy-MM-dd'T'HH:mm:ss.SSSSSS||yyyy-MM-dd'T'HH:mm:ss.SSSSSS'Z'||yyyy-MM-dd'T'HH:mm:ss'Z'||yyyy-MM-dd'T'HH:mm:ssZ||yyyy-MM-dd'T'HH:mm:ssZZ'Z'||yyyy-MM-dd'T'HH:mm:ss.SSSZZ||yyyy-MM-dd'T'HH:mm:ss||yyyy-MM-dd' 'HH:mm:ss||yyyy-MM-dd' 'HH:mm:ss.SSSSSS||mm/dd/yy' 'HH:mm:ss",
```
Comment (dimagimon) [inline corehq/pillows/mappings/user_mapping.py]: E501 line too long (347 > 115 characters)

File: corehq/pillows/mappings/user_mapping.py [hunk 3]
```diff
@@ -56,7 +65,7 @@
                 'email_opt_out': {'type': 'boolean'},
                 'analytics_enabled': {'type': 'boolean'},
                 'eulas': {'dynamic': False,
-                          'properties': {'date': {'format': DATE_FORMATS_STRING,
+                          'properties': {'date': {'format': "yyyy-MM-dd||yyyy-MM-dd'T'HH:mm:ssZZ||yyyy-MM-dd'T'HH:mm:ss.SSSSSS||yyyy-MM-dd'T'HH:mm:ss.SSSSSS'Z'||yyyy-MM-dd'T'HH:mm:ss'Z'||yyyy-MM-dd'T'HH:mm:ssZ||yyyy-MM-dd'T'HH:mm:ssZZ'Z'||yyyy-MM-dd'T'HH:mm:ss.SSSZZ||yyyy-MM-dd'T'HH:mm:ss||yyyy-MM-dd' 'HH:mm:ss||yyyy-MM-dd' 'HH:mm:ss.SSSSSS||mm/dd/yy' 'HH:mm:ss",
```
Comment (dimagimon) [inline corehq/pillows/mappings/user_mapping.py]: E501 line too long (365 > 115 characters)

File: corehq/pillows/mappings/user_mapping.py [hunk 4]
```diff
@@ -71,28 +80,11 @@
                 'is_staff': {'type': 'boolean'},
                 'is_superuser': {'type': 'boolean'},
                 'language': {'type': 'string'},
-                'last_login': {'format': DATE_FORMATS_STRING,
+                'last_login': {'format': "yyyy-MM-dd||yyyy-MM-dd'T'HH:mm:ssZZ||yyyy-MM-dd'T'HH:mm:ss.SSSSSS||yyyy-MM-dd'T'HH:mm:ss.SSSSSS'Z'||yyyy-MM-dd'T'HH:mm:ss'Z'||yyyy-MM-dd'T'HH:mm:ssZ||yyyy-MM-dd'T'HH:mm:ssZZ'Z'||yyyy-MM-dd'T'HH:mm:ss.SSSZZ||yyyy-MM-dd'T'HH:mm:ss||yyyy-MM-dd' 'HH:mm:ss||yyyy-MM-dd' 'HH:mm:ss.SSSSSS||mm/dd/yy' 'HH:mm:ss",
```
Comment (dimagimon) [inline corehq/pillows/mappings/user_mapping.py]: E501 line too long (346 > 115 characters)

**Comment by benrudolph** [comment]
can we keep the ES indexes and just revert teh app status logic?

**Review by benrudolph — APPROVED** [review]
the tests will fail on the es index tests

**Review by benrudolph — APPROVED** [review]

**Comment by benrudolph** [comment]
(merging because it's a revert and needed for deploy)

**Pull request merged**